metacat-common-server/src/main/java/com/netflix/metacat/common/server/util/DataSourceManager.java

```diff
@@ -50,13 +50,13 @@ public static DataSourceManager get() {
     /**
      * Initialize a data source and store it.
      *
-     * @param catalogName catalog name
+     * @param name catalog name
      * @param properties  properties
      * @return DataSourceManager
      */
-    public DataSourceManager load(final String catalogName, final Map<String, String> properties) {
-        if (dataSources.get(catalogName) == null) {
-            createDataSource(catalogName, properties);
+    public DataSourceManager load(final String name, final Map<String, String> properties) {
+        if (dataSources.get(name) == null) {
```

Comment:
what is this name? If this is not 'catalogName' anymore, we should change the comments 
`* @param name catalog name` to the right one.

Reply:
Can be anything, this is a generic class. Consumers can use any name - catalog name or catalog shard name.